lib/sample.report-config.json

```diff
@@ -1,14 +1,18 @@
 [
   {
-    "name": "Enrolled",
-    "reportId": 9080
+    "reportId": 10908,
+    "title": "Eligible and enrolled in placebo adherence run-in (Enrolled/Consented)",
+    "strategy": "total"
   },
   {
-    "name": "Discontinued Prior to Randomization",
-    "reportId": 9082
+    "reportId": 10909,
+    "title": "Excluded (Discontinued)",
+    "strategy": "itemized",
+    "bucket-by": "dsp_stop_reason"
```

Comment:
I'll need to modify the `REDCap::getReport` function in REDCap to support this strategy. The `DataExport::doReport` function takes several arguments that `getReport` hard-codes. I need access to exporting labels otherwise the values for `bucket-by` are the raw values. For example: `{"screen_id":"1001","redcap_event_name":"exit_arm_1","site":"","dsp_stop_reason":"6","dsp_last_visit_name":"1"}`

I updated it earlier and it's a pretty simple fix. `getReport` should probably take the same arguments as `doReport`.